cryptol.cabal

```diff
@@ -264,6 +264,9 @@ executable cryptol
       ld-options: -static -pthread
       ghc-options: -optl-fuse-ld=bfd
 
+  if flag(ffi)
+      ld-options: -Wl,rpath,$ORIGIN
```

Comment:
I think this should be `-rpath=$ORIGIN`. The `-Wl` part is a prefix required by C compilers that direct them to pass the things following it to the linker, but since `ld-options` passes the options directly to the linker, we don't need the `-Wl` part.

Reply:
From running cabal with `-v` it seems like what `ld-options` does is pass it prepended with `-optl` to GHC, which then passes the rest of it to the C compiler, so I think we still need the `-Wl,`. I did forget to put `-` before `rpath` though.

Reply:
Ah, I see. My mistake!